### ISSUE TITLE
Could you add Gemeente Arnhem (https://www.afvalwijzer-arnhem.nl/)

### ISSUE THREAD

**Issue by pascalsaul** [pippyn/Home-Assistant-Sensor-Afvalbeheer#103]
I see Suez is listed but not Gemeente Arnhem or as generic. Could you add https://www.afvalwijzer-arnhem.nl/

https://postcodebijadres.nl/gemeente/arnhem (6800-6846)

**Comment by pippyn** [comment]
I'll have a look.

**Comment by florisi** [comment]
I live in Arnhem too.
When I open the Afvalwijzer, it links to Afvalkalender Suez.
But Suez is not working.
Or I'm doing something wrong.

**Comment by pippyn** [comment]
You can just use a config like this (random address):
```
  - platform: afvalbeheer
    wastecollector: suez
    resources:
      - restafval
    postcode: 6841AE
    streetnumber: 37
```

**Comment by florisi** [comment]
The sensors are created, but they stay empty.
I've seen that in another custom component there was an update for Seuz, something to do with SSL.

**Comment by pippyn** [comment]
Can you give me a link?

**Comment by florisi** [comment]
https://github.com/heyajohnny/afvalinfo

See the first file (Fix SSL Suez)

**Comment by pippyn** [comment]
Fixed in v4.4.2
